openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -98,6 +97,8 @@ class AccountView(APIView):
             * year_of_birth: The year the user was born, as an integer, or null.
             * account_privacy: The user's setting for sharing her personal
               profile. Possible values are "all_users" or "private".
+            * has_accomplishments: Signals whether badges are enabled on the
```

Comment:
To support the UI requirements, `has_accomplishments` should indicate whether or not the user has actually earned any badges.